alpine-chroot-install

```diff
@@ -165,7 +165,7 @@ wgets() (
 		&& rm -f "${url##*/}" \
 		&& download_url "$url" \
 		&& echo "$sha256  ${url##*/}" | sha256sum -c
-)
+}
```

Comment:
Parenthesis are used here intentionally, to execute it in a subshell, so `cd` will not affect the rest of the script.